.circleci/config.yml

```diff
@@ -26,8 +26,13 @@ jobs:
               tar xzf google-cloud-sdk-*.tar.gz
               google-cloud-sdk/install.sh -q
               google-cloud-sdk/bin/gcloud components install beta cloud-datastore-emulator -q
-              google-cloud-sdk/bin/gcloud config set project testing
```

Comment:
Why did we do this?

Reply:
No idea..

Reply:
It looks like we have to set it to use the datastore emulator

Reply:
That's strange because I don't need this locally and CI passed for your previous commit. Maybe just legacy code.

Reply:
Maybe you have to set the project to something. Because in the other commit I set the project to flyte-oss-test but now I don't

Reply:
Then let's set it to the real thing `styx-oss-test`?